build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1690,4 +1690,18 @@ message RequestMetadata {
   // An identifier to tie multiple tool invocations together. For example,
   // runs of foo_test, bar_test and baz_test on a post-submit of a given patch.
   string correlated_invocations_id = 4;
+
+  // A one-word description of the kind of action, for example, CppCompile or GoLink.
```

Comment:
Do we intend to enforce this as being one word? Or can we just say "brief" (which is unenforceable, but at least it's vague enough to make it clear that it won't be formally enforced).

Reply:
Changed to "brief" :)